style.css

```diff
@@ -1281,6 +1276,182 @@ form {
   }
 }
 
+/* Popup section */
+
+.popup {
+  display: flex;
+  backdrop-filter: blur(4px);
+  align-items: center;
+  justify-content: center;
+  margin: auto 2.4rem;
+  width: 100%;
+  height: 100vh;
+  z-index: 1001;
+  top: 0;
+  position: fixed;
+}
+
+.popup-holder {
+  position: fixed;
+  padding: 1.5rem;
+  overflow: hidden;
+  max-width: 75%;
+  width: 100%;
+  height: 90vh;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  background: #dfe1e6;
```

Comment:
- [ ] The background color of popup modal is `#fff` in the figma design. kindly consider  changing this to make your popup modal identical to the figma design because it is a requriement.
- [ ] Also there is some extra padding-top and padding-bottom in your modal while in the figma design the top and bottom padding is less. kindly adjust that to make your modal identical to the figma design. you can check the difference in the following pics.
your design
<img width="892" alt="Screenshot 2023-06-28 at 9 29 44 PM" src="https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/96683843/793e84d7-d67a-4e82-b8a8-9ebfc77b94be">
<img width="489" alt="Screenshot 2023-06-28 at 9 30 00 PM" src="https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/96683843/fcb3b507-beca-472d-858d-48c28c125dda">

figma design
<img width="229" alt="Screenshot 2023-06-28 at 9 30 10 PM" src="https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/96683843/76c84074-2cae-4042-900b-708eed9d8b6d">

<img width="708" alt="Screenshot 2023-06-28 at 9 28 29 PM" src="https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/96683843/2c0da009-a928-4e36-a534-e35e66142f46">